markdown/ch5.markdown

```diff
@@ -38,9 +38,11 @@ def variation_of_information(x, y):
     Px_inv = sparse.diags(invert_nonzero(px))
     Py_inv = sparse.diags(invert_nonzero(py))
 
-    # then, compute the entropies
-    hygx = px @ xlog1x(Px_inv @ Pxy).sum(axis=1)
-    hxgy = xlog1x(Pxy @ Py_inv).sum(axis=0) @ py
+    # then, compute the entropies, using dot-product for the sum
+    ex = np.ones_like(px)
```

Comment:
*sigh* in retrospect you're right. From our own preface:

> it is often *concise* (but not so concise as to be obscure)

=P I really like the symmetry of the approach, and I think it will actually be *more* readable for linear algebra gurus, but that is a minority of our readers. Reverting.